junit_tests/testTimeDelta.java

```diff
@@ -0,0 +1,423 @@
+import org.junit.Test;
+import org.python.stdlib.datetime.TimeDelta;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.python.types.Bool.getBool;
+import static org.python.types.Int.getInt;
+
+public class testTimeDelta {
```

Comment:
Java Naming Convention: Uppercase class names (maybe `TimeDeltaTest`?)

Reply:
Fixed in 6296ecf401e592553c038b6f16ab6e1e2299af73! 👍